### PR TITLE
Update Teads amp-ad tag and readme

### PR DESCRIPTION
**Pull request by RonanDrouglazet**
- update Teads tag to support custom json tag (internal use only so not in the README)

- update README to fit our guidelines

### CODE REVIEW

**Comment by RonanDrouglazet** [comment]
@googlebot CLA Signed on behalf of Corporation "Teads"

File: ads/teads.md [hunk 1]
```diff
@@ -19,20 +19,17 @@ limitations under the License.
 ## Example
 
 ```html
-<amp-ad width=300 height=220
+<amp-ad width=300 height=1
```
Comment (cramforce) [inline ads/teads.md]: Is this working well for you?

Is `layout=responsive` working OK with such an extreme aspect ratio?

**Comment by RonanDrouglazet** [comment]
Hi @cramforce ! 

More or less... It's not the best behavior for us but currently, but we've got no other option.

Our initial issue was the "blank slot" effect when noContentAvailable was called, resize denied and no fallback as reported here https://github.com/ampproject/amphtml/issues/3182

Fallback is not currently a solution for us, and publishers were not willing to use such feature. Blank slots where considered bugs.

BUT we can't only update the responsive ratio value, and you are not willing to support it as mentioned here https://github.com/ampproject/amphtml/issues/3227

When we ask for a resize to open our slot, responsive is broken with fixed css width / height value.
You suggested to ask again for resize on orientation change but it's not working well, 75% of the time its denied (this PR could allow me to create a sample to talk about this with you). Currently, our slot are not responsive, so in landscape, we keep portrait's width etc.. hard to find this value when we are loaded on landscape... 

Also, we found some bugs on initialIntersection.rootBounds under iphone 5 / 6 etc (safari): screen values are not correct sometimes which make it very difficult for us to calculate our slot height and width.

As I said, this sounds like a workaround but it doesn't seem like the best solution. 

The best solution for us would be :

-> a new method contentAvailable() like noContentAvailable but in the other way (only open a slot IF contentAvailable AND resizing rules ok) 
OR
-> give us the ability to update the responsive ratio and not the css size

**Comment by cramforce** [comment]
Did you file issues about the iPhone bugs you were seeing?

If this is working for you, it is fine by me.

**Comment by RonanDrouglazet** [comment]
Not yet, this PR will allow us to create a sample case and be more accurate in the communication with you on this subject

**Comment by cramforce** [comment]
LGTM

**Pull request merged**